nflgame/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.1b1'
+__version__ = '2.0.1a6'
```

Comment:
Looks like your `__version__` may have reverted to an older numbering.

Reply:
Huh. Weird.

Fixed to be higher.